tensorboard/webapp/runs/store/runs_reducers.ts

```diff
@@ -171,6 +185,10 @@ const dataReducer: ActionReducer<RunsDataState, Action> = createReducer(
     return {
       ...state,
       selectionState: nextSelectionState,
+      shouldAutoSelectRuns: new Map([
+        ...state.shouldAutoSelectRuns,
+        [stateKey, nextValue],
```

Comment:
Similar to this, shouldn't we set the `shouldAutoSelectRuns` to false when one is toggled off? 

If the goal is to be consistent with the Scalars plugin, then we should not select by default when at least one is toggled off by user.